environment/terraform.tfvars.json

```diff
@@ -68,7 +71,8 @@
       "symfony_env": "test",
       "db_subnet_group": "private",
       "ec_subnet_group": "private",
-      "state_source": "preproduction"
+      "state_source": "preproduction",
+      "always_on": true
```

Comment:
Is not feasible for `master` and `preproduction` to have `"always_on": false`?

Reply:
Eventually it will be, I've got [another ticket](https://opgtransform.atlassian.net/browse/DDPB-3242) in for that.

We can't do it (easily) in this PR because it requires in-place replacement of the database. For preproduction/training we'll have to do a proper migration to avoid data loss (and have [upgraded to Postgres 10](https://opgtransform.atlassian.net/browse/DDPB-3228)).

Also, Serverless requires Postgres 10 but production currently uses Postgres 9. Keeping master on Postgres 9 for now ensures the tests are run against the correct DB version at some point in the pipeline.